is-authenticator-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -87,4 +178,59 @@
             </plugin>
         </plugins>
     </build>
-</project>
+    <repositories>

```

Comment:
Since we are releasing the complete repo this is not required and will be taken from parent